packages/@aws-cdk/pipelines/README.md

```diff
@@ -1274,6 +1274,22 @@ encryption key policy for the artifacts bucket may have a statement that looks l
 
 Any resource or policy that references the qualifier (`hnb659fds` by default) will need to be updated.
 
+### This CDK CLI is not compatible with the CDK library used by your application
```

Comment:
Can we do better? Why not always use the latest CLI (within the same major version)? Our compatibility model should support this.

Reply:
That is the default, but people still have the option of specifying an explicit version if they want to. It's risk mitigation.

We've had some bugs in the CLI that break people's pipelines, it's good if they have an "out" to continue their deployments, or even not be affected in the first place.

Reply:
OK. Is it clear from our docs that we do not recommend to use an explicit CLI version? Maybe it's worth mentioning in this section as well that the recommended practice is to avoid pinning the CLI version since new CLIs are always expected to be compatible with older version of the CDK.

Reply:
I can add some phraseology to that effect.